LPS-205004 Add test for configuring editable link with custom URL

sandrodw3 opened the pull request.

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

sandrodw3 commented:
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

sandrodw3 commented:
Pull request submitted to https://github.com/ealonso/liferay-portal/pull/2085. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

sandrodw3 committed:
LPS-205004 Add HeadlessSiteApiHelper with utilities to create and delete a site

sandrodw3 committed:
LPS-205004 Add HeadlessDeliveryApiHelper with utility to create a page

sandrodw3 committed:
LPS-205004 Add test for checking configuration sections visibility

sandrodw3 commented:
ci:test:playwright

sandrodw3 commented:
ci:test:playwright

sandrodw3 committed:
LPS-205004 Add util to generate random id

sandrodw3 committed:
LPS-205004 Add PageDefinition and PageElement types

sandrodw3 committed:
LPS-205004 Add utilities to get page and fragment definitions

ruben-pulido committed:
LPS-205209 Add delete endpoint to support deleting a site by external reference code

ruben-pulido committed:
LPS-205209 Add delete endpoint to support deleting a site by site id

ruben-pulido committed:
LPS-205209 buildREST

ruben-pulido committed:
LPS-205209 Implement deleteSiteByExternalReferenceCode

ruben-pulido committed:
LPS-205209 Implement deleteSite

ruben-pulido committed:
LPS-205209 Override testDeleteSiteByExternalReferenceCode_addSite method

ruben-pulido committed:
LPS-205209 Override testDeleteSite_addSite method

ruben-pulido committed:
LPS-205209 Make error messages consistent

ruben-pulido committed:
LPS-205209 NoSuchGroupSiteExceptionMapper should account for the case where message contains external reference code instead of groupKey

ruben-pulido committed:
LPS-205209 Add test deleting site with nonexistent external reference code

ruben-pulido committed:
LPS-205209 Add test deleting site with nonexistent site ID

ruben-pulido committed:
LPS-205209 Adapt expected error message to the one returned by com.liferay.portal.vulcan.internal.jaxrs.param.converter.provider.SiteParamConverterProvider.

Note that this error message needs to be adapted within "https://liferay.atlassian.net/browse/LPS-181443 - Unify pattern for Headless API error messages" so it follows the agreed error message guidelines.

sandrodw3 committed:
LPS-205004 Update package-lock.json
